src/VPolygon.jl

```diff
@@ -553,3 +553,52 @@ function translate(P::VPolygon{N}, v::AbstractVector{N}) where {N<:Real}
                                 "set by a $(length(v))-dimensional vector"
     return VPolygon([x + v for x in vertices_list(P)])
 end
+"""
+    minkowski_sum(P::VPolygon{N}, Q::VPolygon{N}) where {N<:Real}
+
+The Minkowski Sum of two polygon in vertex representation.
+
+### Input
+
+- `P` -- polygon in vertex representation
+- `Q` -- another polygon in vertex representation
+
+### Output
+
+A polygon in vertex representation.
+
+### Algorithm
+
+We treat each edge of the polygons as a vector, attaching them in polar order
+(attaching the tail of the next vector to the head of the previous vector). The
+resulting polygonal chain will be a polygon, which is the Minkowski sum of the given polygons.
+"""
+
+function minkowski_sum(P::VPolygon{N}, Q::VPolygon{N}) where {N<:Real}
+    vlP = vertices_list(P)
+    vlQ = vertices_list(Q)
+    mP = length(vlP);
+    mQ = length(vlQ);
+    i = 1
+    k = 1
+    j = 1
+    R = Vector{Vector{Float64}}(undef, mP+mQ)
+    fill!(R, N[0, 0])
+    while i <= size(R, 1)
+        OP = vlP[(k%mP+1)]-vlP[(k-1)%mP+1]
```

Comment:
it confuses me to call this variable "OP", since it does not represent the segment from the origin (O) to some point in the polygon `P`, it is a line segment from point `P₁` and `P₂`:

```julia
P₁, P₂ = vlP[(k-1)%mP+1], vlP[(k%mP+1)]
P₁P₂ = P₂ - P₁
```